src/v2/BaseLoginRouter.js

```diff
@@ -53,6 +62,7 @@ export default Router.extend({
 
     this.settings = new Settings(_.omit(options, 'el', 'authClient'), { parse: true });
     this.settings.setAuthClient(options.authClient);
+    this.settings.set('proxyIdxResponse', options.proxyIdxResponse);
```

Comment:
i think adding a local property in src/models/Settings.js would make more sense.. instead of setting this here. @haishengwu-okta can confirm once.